cppcache/integration-test/CMakeLists.txt

```diff
@@ -114,6 +115,7 @@ endif()
   target_link_libraries(${TEST}
     PRIVATE
       ACE::ACE
+      Boost::boost
```

Comment:
Same.

Reply:
After changing CacheStatistics this is not needed anymore. Thanks!

Reply:
After carefully looking at this I've noticed that new integration tests also include Boost library.
Thing is symbols are not needed from Boost library for integrations test, but compilation flags from the library.
Thing is in order to ignore clang-tidy checks for boost you need to include it as a system library, which is what's achieved by "linking" the library.